src/BootstrapButtonGroupField.php

```diff
@@ -34,18 +39,18 @@ public function getOptions()
 				'Selected' => $this->Value() == $val
 			)));
 		}
-		
-		return $options;		
+
+		return $options;
 	}
 
 
 
 	public function Field($attributes = array ())
 	{
 		Requirements::javascript(BOOTSTRAP_FORMS_DIR."/javascript/bootstrap_forms.js");
```

Comment:
Use module reference here: unclecheese/boostrapforms: file.js`

Reply:
Fixed in https://github.com/unclecheese/silverstripe-bootstrap-forms/pull/56/commits/034d9e8b441a32fa13ca2271f0ec04a0e6c71525